src/fluree/db/query/exec/eval.cljc

```diff
@@ -789,159 +808,92 @@
-                      cycle cycle
-                      interleave interleave
-                      interpose interpose
-                      str str
-                      subs subs
-                      re-find re-find
-                      re-matches re-matches
-                      re-pattern re-pattern
-                      re-seq re-seq
-                      inc inc
-                      dec dec
-                      + +
-                      - -
-                      * *
-                      / /
-                      quot quot
-                      rem rem
-                      mod mod
-                      abs abs
-                      max max
-                      min min
-                      rand rand
-                      rand-int rand-int
-                      compare compare
-                      = =
-                      not= not=
-                      < <
-                      > >
-                      <= <=
-                      >= >=
-                      zero? zero?
-                      pos? pos?
-                      neg? neg?
-                      even? even?
-                      odd? odd?
-                      true? true?
-                      false? false?
-                      identity identity
-                      constantly constantly
-                      comp comp
-                      complement complement
-                      partial partial
-                      memoize memoize
-                      atom atom
-                      deref deref
-                      reset! reset!
-                      swap! swap!
-                      compare-and-set! compare-and-set!
-                      meta meta
-                      with-meta with-meta
-                      name name
-                      namespace namespace
-                      symbol symbol
-                      keyword keyword
-                      apply apply
-                      pr-str pr-str
-                      prn-str prn-str
-                      println println
-                      print print
-                      newline newline}
-
-           ;; Resolve core functions
-           resolved-core-fns (reduce (fn [acc [k v]]
-                                       (if-let [resolved (resolve v)]
-                                         (assoc acc k @resolved)
-                                         acc))
-                                     {}
-                                     core-fns)
-
-           ;; Add platform-specific functions
-           resolved-core-fns (assoc resolved-core-fns
-                                    'format #?(:clj format :cljs str))
-
-           ;; Build user namespace by merging eval namespace with qualified -and/-or
-           user-ns-fns (assoc eval-ns-fns
-                              'fluree.db.query.exec.eval/-and -and-fn
-                              'fluree.db.query.exec.eval/-or -or-fn)]
+                     'const/iri-id const/iri-id
```

Comment:
Same deal here, you don't need `const/iri-id` since you've already got `iri-id`.